doc/api/fs.md

```diff
@@ -3077,6 +3077,74 @@ error raised if the file is not available.
 To check if a file exists without manipulating it afterwards, [`fs.access()`]
 is recommended.
 
+For example, given the following folder structure:
+
+-txtDir
```

Comment:
It seems we need backticks around this fragment, otherwise, it is rendered as one line.